python/ray/tune/examples/pbt_convnet_example.py

```diff
@@ -133,7 +133,7 @@ def stop_all(self):
 
     best_trial = analysis.get_best_trial("mean_accuracy")
     best_checkpoint = max(
-        analysis.get_trial_checkpoints_paths(best_trial, "mean_accuracy"))
+        analysis.get_trial_checkpoints_paths(best_trial, "mean_accuracy"), key=lambda x:x[1])
```

Comment:
hmm, this is a little gross - does it make sense to provide a helper function in analysis?

Reply:
I added a new function to do this in Analysis.